### PR TITLE
Custom SSLContext

### PR DESCRIPTION
**Pull request by diegolovison**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,3 +142,21 @@ HTTPS=true mvn quarkus:dev
 as usual - the `HTTPS=true` will use secured connections on the live-reload proxy on port 3000.
 
 When you want to revert back to plain HTTP, run `./disable-https.sh` and drop the `HTTPS=true` env var.
+
+## SSL in `horreum-client`
+You can set a certificate in runtime for `HorreumClient`
+```java
+InputStream fis = new FileInputStream("/path/to/my-cert.pem");
```
Comment (rvansa) [inline README.md:149]: Could you rather move all of this to a utility method next to `sslContext(...)`?

Comment (diegolovison) [reply README.md]: Done.

File: horreum-client/src/main/java/io/hyperfoil/tools/HorreumClient.java [hunk 2]
```diff
@@ -102,24 +113,64 @@ public Builder clientSecret(String clientSecret) {
...
+         * @param certFilePath certificate path
+         * @param type the name of the requested certificate type. See the CertificateFactory section in the Java
+         * Security Standard Algorithm Names Specification for information about standard certificate types.
+         * @param alias the alias name
+         * @param protocol the standard name of the requested protocol. See the SSLContext section in the Java Security
+         * Standard Algorithm Names Specification for information about standard protocol names.
+         *
+         * @return
+         */
+        public Builder sslContext(String certFilePath, String type, String alias, String protocol) {
```
[the remark this answers is not in the document]
Comment (diegolovison) [reply horreum-client/src/main/java/io/hyperfoil/tools/HorreumClient.java]: Done.

**Review by rvansa — APPROVED** [review]

**Comment by rvansa** [comment]
LGTM, thanks!

**Pull request merged**